packages/react-composites/src/composites/CallComposite/ScreenShare.tsx

```diff
@@ -99,7 +100,11 @@ export const ScreenShare = (props: ScreenShareProps): JSX.Element => {
         {videoStream && videoStream.isAvailable && videoStream.renderElement && (
           <Stack horizontalAlign="center" verticalAlign="center" className={aspectRatioBoxStyle}>
             <Stack className={aspectRatioBoxContentStyle}>
-              <VideoTile renderElement={<StreamMedia videoStreamElement={videoStream.renderElement ?? null} />} />
+              <VideoTile
+                displayName={screenShareParticipant?.displayName}
+                isMuted={screenShareParticipant?.isMuted}
+                renderElement={<StreamMedia videoStreamElement={videoStream.renderElement ?? null} />}
```

Comment:
@prprabhu-ms should we do 
renderElement={videoStream.renderElement ? <StreamMedia videoStreamElement={ videoStream.renderElement } />} : undefined here?

I believe it is more aligned with the logic of previous PRs

Here

Reply:
yep, good catch, @PorterNan